src/main/java/com/github/kiulian/downloader/model/search/ItemType.java

```diff
@@ -0,0 +1,8 @@
+package com.github.kiulian.downloader.model.search;
+
+public enum ItemType {
```

Comment:
very general name, maybe name it SearchItemType?

Reply:
Right, let it be `SearchResultItemType`.